### ISSUE TITLE
Initialising CTL with key store is successful with incorrect key store location or incorrect key store password

### ISSUE THREAD

**Issue by dulanjalidilmi** [wso2/product-apim-tooling#651]
**Description:**
Initializing with keys store is successful when incorrect key store location or incorrect key store is provided. 

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by uvindra** [comment]
I believe when running the `apictl secret init` command we are not doing any executions but simply setting up the location of the key store and credentials that will be required to do subsequent encryption. Therefore not sure if we can do such a validation at this stage. @SanojPunchihewa can you comment?

**Comment by SanojPunchihewa** [comment]
> I believe when running the apictl secret init command we are not doing any executions but simply setting up the location of the key store and credentials that will be required to do subsequent encryption.

Yes. In the init command we just save those parameters to a config file. But it's possible to have a validation similar to what we do in `apictl secret create` [[1](https://github.com/wso2/product-apim-tooling/blob/ccf1593585f2993916bc873c1e4476e4eb5b0aa9/import-export-cli/utils/encryptionUtils.go#L156)] here as well.
